wiki/impaled.md

```diff
@@ -125,11 +125,11 @@ It can be enchanted with Unbreaking, Mending and Efficiency, the latter to reduc
 
 #### Can I include this mod in a modpack?
 
-**Yes**, you can. Go ahead, don't bother asking. Please  however provide credit and a link to both the [GitHub repository](https://github.com/Ladysnake/Impaled) and [Curse Forge project page](https://www.curseforge.com/minecraft/mc-mods/Impaled).
+**Yes**, you can. Go ahead, don't bother asking. Please  however provide credit and a link to both the [GitHub repository](https://github.com/ladysnake/impaled) and [Curse Forge project page](https://www.curseforge.com/minecraft/mc-mods/impaled).
```

Comment:
The GH link with capital letters is _technically_ the correct one, but GitHub will allow either

Reply:
I wanted it to be consistent between links iirc